api/defaultGroupMembership.go

```diff
@@ -0,0 +1,152 @@
+package api
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/shurcooL/graphql"
+)
+
+type DefaultGroups struct {
+	client *Client
+}
+
+type defaultGroup struct {
+	Users []struct {
+		Username string
+		Id       string
+	}
+}
+
+type defaultGroupData struct {
+	Admin      defaultGroup
+	Eliminator defaultGroup
+	Member     defaultGroup
+}
+
+type searchDomainData struct {
+	Name  string
+	Users []struct {
+		Username string
+		Groups   []struct {
+			DisplayName string
+		}
+	}
+}
+
+type DefaultGroupEnum string
+
+type Groups []DefaultGroupEnum
+
+const (
+	Eliminator DefaultGroupEnum = "Eliminator"
+	Admin      DefaultGroupEnum = "Admin"
+	Member     DefaultGroupEnum = "Member"
+)
+
+func (c *Client) DefaultGroup() *DefaultGroups { return &DefaultGroups{client: c} }
+
+func (d *defaultGroupData) groupsForUser(username string) Groups {
+	groups := []DefaultGroupEnum{}
+
+	for _, a := range d.Admin.Users {
+		if a.Username == username {
+			groups = append(groups, Admin)
+		}
+	}
+
+	for _, a := range d.Eliminator.Users {
+		if a.Username == username {
+			groups = append(groups, Eliminator)
+		}
+	}
+
+	for _, a := range d.Member.Users {
+		if a.Username == username {
+			groups = append(groups, Member)
```

Comment:
You can `break` the loop here as there is no need to check the remaining users.
```suggestion
			groups = append(groups, Member)
			break
```